R/DiseasyActivity.R

```diff
@@ -39,12 +81,24 @@ DiseasyActivity <- R6::R6Class( # nolint: object_name_linter
 
       lgr::without_logging(self$reset_scenario()) # Configure internal matrices to activity_unit dimensions
 
-      # TODO: add documentation here
+      # If the base_scenario is "dk_reference", we load a configured activity scenario into the module
       if (base_scenario == "dk_reference") {
-        self$set_activity_units(mg_activity_units)
-        self$change_activity(mg_reference_scenario)
-        work_risk <- aggregate(faWork ~ date, data = mg_reference_scenario, FUN = mean)
-        self$change_risk(date = work_risk$date, type = "work", risk = work_risk$faWork)
+
+        # Use the Danish activity units
+        self$set_activity_units(dk_activity_units)
+
+        # Use the Danish restrictions
+        self$change_activity(dk_reference_scenario)
+
+        # TODO: Lasse, is this correct?
+        # TODO: wouldn't it be nice if this happened inside $change_activity?
+        # TODO: this way, the `dk_reference_scenario` could be loaded in a single function?
```

Comment:
TODO to be resolved

Reply:
I find it usefull to split the setting of activity units - all that are allowed - and the actual opening and closing of those. If there were more reference scenarios with individual activity units then one could consider making an internal function that sets both the units and the reference changes in activity. That could be next version ...